chapter07/mdw/Ex04.pl

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use 5.010;
+
+foreach (<>) {
+  if ( /\b[^\s\\]*[A-Z]+[^\s\\]*[a-z]+[^\s\\]*\b/ || /\b[^\s\\]*[a-z]+[^\s\\]*[A-Z]+[^\s\\]*\b/ ) {
+ # comment as i have made this more complicated than needed, this will match any word with anything+capital+anything+non-capital+anything or the othe way around
+    print $_
+  }
```

Comment:
That does not do the same thing. This checks for a single word with both a capital and a non-capital, as opposed to checking the line. I was looking to capture each word and then run that command originally but could not figure out how. I considered splitting the input but felt that was not using regex.